rootfs/scheduler/__init__.py

```diff
@@ -568,42 +583,53 @@ def run(self, namespace, name, image, entrypoint, command, **kwargs):
         if unhealthy(response.status_code):
             error(response, 'create Pod in Namespace "{}"', namespace)
 
-        data = ''
-        duration = 30
-        iteration = 1
-        while (iteration < duration):
-            try:
+        labels = {
+            'app': namespace,
+            'type': 'run',
+            'version': kwargs.get('version'),
```

Comment:
That is a label selector which I am using for filtering and if you look at the `POD_TEMPLATE` then `$appversion` is for the templating but the key it self is `version`

I really need to just rename `$version` to `$apiversion` to remove this confusino